lib/virtmanager.pm

```diff
@@ -15,9 +15,11 @@ sub launch_virtmanager() {
     type_string "virt-manager", 50;
     send_key "ret";
     wait_idle;
-    type_password;
-    send_key "ret";
-    wait_idle;
+    if (check_var("FLAVOR", "Server-DVD")) {
```

Comment:
I am still not sure if we need to distinguish FLAVOR here?
@coolo, @dzedro please decide